WORKSPACE

```diff
@@ -265,6 +265,12 @@ new_go_repository(
 )
 
 new_go_repository(
+    name = "io_k8s_apimachinery",
+    commit = "d3c1641d0c440b4c1bef7e1fc105f19f713477e0",
+    importpath = "k8s.io/apimachinery",
```

Comment:
`client-go` we use it too old to include this library. Is there a potential problem using a newer version of api machinery? I think we need to update client-go library and do a big import rename.